hkdfchacha20poly1305/hkdfchachapoly_test.go

```diff
@@ -1,17 +1,67 @@
-package xchacha20poly1305
+package hkdfchacha20poly1305
 
 import (
-	"encoding/hex"
+	"bytes"
+	cr "crypto/rand"
+	mr "math/rand"
+	"testing"
 )
 
-func toHex(bits []byte) string {
-	return hex.EncodeToString(bits)
-}
+// Copied from https://github.com/golang/crypto/blob/master/chacha20poly1305/chacha20poly1305_test.go#L69
```

Comment:
Is there some official way I need to document this?

Reply:
We'll have to make sure we follow the requirements of this license and if we can mix with what we use etc:
https://github.com/golang/crypto/blob/a8fb68e7206f8c78be19b432c58eb52a6aa34462/chacha20poly1305/chacha20poly1305_test.go#L1-L3